### ISSUE TITLE
Missing .ioc

### ISSUE THREAD

**Issue by gargas-arl** [tjaekel/STM32U5xx_USB_ECM#2]
Looks like only the STM32U5xx_USB_ECM.ioc was provided, but it seems to be geared at the non 144 pin version of the U575 only. Do you have a representative .ioc file you could share so others could build projects on top of it?

**Comment by tjaekel** [comment]
Yes, this is correct: just an .ICO file for a low pin count U575, not for a larger one, not for an U5A5 one (where I use the code).
Why? Usually I do not create a full blown STM32 CubeMX project: often, I just configure the minimum pins (here, the USB pins) and let generate an "empty" project, to download all the drivers I need.
I never add all the pins for a final project on CubeMX, instead: I add in C-code, after a basic project as template was generated.
You can generate your own .ICO file by using CubeMX and setup it up with your right MCU.
I do not like to start updating on C-code from CubeMX, when I have already added several C-code to my STM32 IDE project: it can corrupt the code, remove my manually added code. So, such a CubeMX .ICO file there left is just by "mistake" (not by the intention to provide a full blown CubeMX for this project).
You can build the project via STM32 IDE: there is also a macro to toggle between U575 vs. U5A5. CubeMX (.ICO) would not do really so much, just creating a base template and to make sure to have all the driver and HAL code. Later, when progressing by adding code - starting again from CubeMX is "too risky" for me (e.g. sometimes I had to extend or modify the HAL drivers: CubeMX could overwrite and get rid of my fixes and changes).
The drivers (HAL, USB stack, RTOS/AzureOS) should be the same for the entire family (e.g. U575 vs. U5A5 and different pin counts on parts).

Please, take it as a STM32 IDE project, not as a CubeMX project.
